src/rpc/quorums.cpp

```diff
@@ -730,6 +730,9 @@ static UniValue quorum_getdata(const JSONRPCRequest& request, const LLMQContext&
         } else {
             throw JSONRPCError(RPC_INVALID_PARAMETER, "proTxHash missing");
         }
+    } else if (!request.params[4].isNull()) {
+        // Require no proTxHash otherwise
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Should not specify proTxHash");
```

Comment:
Could you please explain why?

Reply:
Because in this case we ask for a quorum wide data only (`QUORUM_VERIFICATION_VECTOR`) and not for a data about one specific MN, so `proTxHash` is not used in any way in this case and should not be provided. If it still was provided then maybe user doesn't quite understand what he is doing exactly or maybe he made a typo in `dataMask`.

Reply:
Got it. Thanks!

Reply:
I think this would be a breaking change, no?